File: C++/Estrutura Sequencial/Idades.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;

int main()
{

    string nome1, nome2;
    int idade1, idade2;
    double media;

    cout << "Dados da primeira pessoa: " << endl;
    cout << "Nome ";
    getline(cin, nome1);
    cout << "Idade: ";
    cin >> idade1;

    cout << "Dados da segunda pessoa " << endl;
    cin.ignore(INT_MAX, '\n');
    cout << "Nome: ";
    getline(cin, nome2);
    cout << "Idade: ";
    cin >> idade2;

    media = (double)(idade1 + idade2) / 2;

    cout << fixed << setprecision(1);
    cout << "A media de idade de " << nome1 << " e " << nome2 << " eh de " << media << " anos " << endl;

    return 0;
}

File: Python/Estrutura Sequencial/Idades.py
nome1: str; nome2: str
idade1: int; idade2: int; soma: int
media: float


print(f"Dados da primeira pessoa: ")
nome1 = str(input("Nome: "))
idade1 = int(input("Idade: "))

print(f"Dados da segunda pessoa: ")
nome2 = str(input("Nome: "))
idade2 = int(input("Idade: "))

soma = idade1 + idade2
media = soma/2

print(f"A idade media de {nome1} e {nome2} é de {media} anos ")